scripts/Import.js

```diff
@@ -86,13 +95,14 @@ class ImportScript extends Base {
     // them, batch them, index them in Algolia) while imposing a queue so that only so many
     // indexing threads will be run in parallel
     if (filenames.length <= 0) {
-      console.log('Done reading files\n');
+      console.log('Done reading files');
       return;
     }
     // Start new file read stream
     // Note: filenames is a reference to the mutable class instance variable this.filenames
     const filename = filenames.pop();
     const file = `${this.directory}/${filename}`;
+    const isCsv = filename.split('.').pop() === 'csv';
```

Comment:
Just in case you have the need later on, there are libraries that can detect the actual file types based on the content not the extension, so that no matter what you get for input you can know what it is:
- https://github.com/mscdex/mmmagic#readme
- https://github.com/sindresorhus/file-type

Reply:
Cool. I like the look of https://github.com/sindresorhus/file-type, I think I'll be looking to re-implement with that in the future..